packages/badge/Badge.test.tsx

```diff
@@ -47,11 +59,17 @@ describe( Badge.is, () => {
       </div>
     ).waitFor(() => true ).then((wrapper) => {
       const badges = wrapper.all(Badge);
-      return  badges[0].has('c-badge__brand') &&
-              badges[1].has('c-badge__info') &&
-              badges[2].has('c-badge__warning') &&
-              badges[3].has('c-badge__success') &&
-              badges[4].has('c-badge__error');
+      const actualBrand = badges[0].node.shadowRoot.querySelectorAll('.c-badge--brand').length;
```

Comment:
mmm before my next  comment try using `one`, it should walk inside shadowRoot

maybe we should create a private function to access shadowRoot instead of repeating zilion times

smthng like `getShadowRoot(element){}`